tests/queries/0_stateless/02813_series_decompose.sql

```diff
@@ -0,0 +1,21 @@
+-- Tags: no-fasttest, no-cpu-aarch64
```

Comment:
`no-fasttest` is obsolete by now and can be removed as we are no longer using a submodule